mpt/impl/imap-mailbox/core/pom.xml

```diff
@@ -66,6 +66,10 @@
             <artifactId>james-server-mailbox-adapter</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.apache.james</groupId>
+            <artifactId>metrics-logger</artifactId>
```

Comment:
No, metrics-logger is one implementation of metrics-api.
In MPT, i've bounded the simple, and only one yet, implementation.